docs/core/get_started/installation.md

```
@@ -1,162 +1,57 @@
 
-## Virtual environment (recommended)
+## 3. Start asking questions
 
-Keep wren-engine and its dependencies isolated from your system Python:
+Once onboarding finishes, just ask your agent business questions in natural language. The agent uses Wren AI's semantic layer to resolve schema, recall similar past queries, and generate accurate SQL.
 
-```bash
-python3 -m venv ~/.venvs/wren
-source ~/.venvs/wren/bin/activate
-pip install "wren-engine[postgres,main]"
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)